src/driver-marsgaming/marsgaming-protocol.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+#include "marsgaming-buttons.h"
+#include "marsgaming-leds.h"
+
+enum marsgaming_report_type {
+	MARSGAMING_MM4_REPORT_TYPE_UNKNOWN_1 = 0x01,
+	MARSGAMING_MM4_REPORT_TYPE_WRITE = 0x02,
+	MARSGAMING_MM4_REPORT_TYPE_READ = 0x03,
+	MARSGAMING_MM4_REPORT_TYPE_UNKNOWN_4 = 0x04,
+	MARSGAMING_MM4_REPORT_TYPE_UNKNOWN_6 = 0x06
+};
+
+struct marsgaming_report_resolution_info {
```

Comment:
All the `_info`  sufixes are a mouthful, maybe they are not needed?
(No need for action, just a suggestion)

Reply:
I could replace them with `_payload` or `_contents`, but I want to make it explicit that it's not the whole report, just the data relating to resolutions. I'm choosing to leave it as is.